test/Table_Tests/src/In_Memory/Fan_Out_Spec.enso

```diff
@@ -14,7 +15,7 @@ add_specs suite_builder =
             t = Table.new cols
             expected_rows = [[0, "a", "c", Nothing], [1, "c", "d", "ef"], [2, "gh", "ij", "u"]]
             expected = Table.from_rows ["foo", "bar 1", "bar 2", "bar 3"] expected_rows
-            t2 = Fan_Out.fan_out_to_columns t "bar" (_.split "b")
+            t2 = Fan_Out.fan_out_to_columns t "bar" (_.split "b") Columns_To_Add.All_Columns 
```

Comment:
As with the other tests, let's keep the old ones to make sure we have kept backward compatibility and just add additional tests for the new API.

Reply:
This was an internal helper, so I think breaking compatibility there is not a problem, right?

Reply:
Yep. Sorry missed this was internal.